internal/testing/prometheus_stuffs_node_labels.yaml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Why do you need these extra prom deployments? Can't we leverage the existing one?

Reply:
Similar to below, I wanted to be able to run different local-test* kind clusters independently.  I needed different prom deployments in order to be able to bind to different ports in docker.